styles/globals.css

```diff
@@ -25,6 +25,14 @@ h6 {
   @apply tracking-tight;
 }
 
+input[type="checkbox"]:focus-visible,
+[role="checkbox"]:focus-visible {
```

Comment:
The [Radix checkbox](https://www.radix-ui.com/primitives/docs/components/checkbox) is a button with a role of checkbox.